poly/src/polynomial/mod.rs

```diff
@@ -26,7 +26,6 @@ pub trait Polynomial<F: Field>:
     + CanonicalSerialize
     + CanonicalDeserialize
     + for<'a> AddAssign<&'a Self>
-    + for<'a> AddAssign<(F, &'a Self)>
```

Comment:
ditto

Reply:
I think this was seen as a false positive with `for<'a> AddAssign<&'a Self>`, just fixed